test/Microsoft.Health.Fhir.Shared.Tests.Integration/Persistence/SqlServerFhirStorageTestHelper.cs

```diff
@@ -50,6 +50,7 @@ public class SqlServerFhirStorageTestHelper : IFhirStorageTestHelper, ISqlServer
             _searchParameterDefinitionManager = searchParameterDefinitionManager;
             _sqlServerFhirModel = sqlServerFhirModel;
             _sqlConnectionFactory = sqlConnectionFactory;
+            _searchParameterDefinitionManager.StartAsync(CancellationToken.None).Wait();
```

Comment:
+1 nice work around moving this to the constructor

Reply:
I think it's actually somewhat antipattern to switch from async/await to Wait method.
Can be wrong but Wait() has blocking behavior while await is not.

---
In reply to: [533650616](https://github.com/microsoft/fhir-server/pull/1449#discussion_r533650616) [](ancestors = 533650616)

Reply:
In generate, this is definitely true.  In this case downstream test resources need this method to finish, and I don't think the blocking behavior will negatively affect the test execution.